test/e2e/autoscaler.go

```diff
@@ -71,11 +72,13 @@ type AutoscalerSpecInput struct {
 // being deployed in the workload cluster.
 func AutoscalerSpec(ctx context.Context, inputGetter func() AutoscalerSpecInput) {
 	var (
-		specName         = "autoscaler"
-		input            AutoscalerSpecInput
-		namespace        *corev1.Namespace
-		cancelWatches    context.CancelFunc
-		clusterResources *clusterctl.ApplyClusterTemplateAndWaitResult
+		specName           = "autoscaler"
+		mpNodeGroupMinSize = "1"
```

Comment:
Let's add a comment to document that we have to use 1 because the MachinePool is initially created without the annotations and the replicas field set and the MachinePool webhook will then set 1 in that case